DevelopGitHub/GitHubExtension/GithubExtension.Extensions/UseOwinContextInjectorExtension.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Owin;
+using SimpleInjector;
+using SimpleInjector.Extensions.ExecutionContextScoping;
+
+namespace GithubExtension.Extensions
+{
+    public static class UseOwinContextInjectorExtension
+    {
+        public static void UseSimpleInjectorContext(this IAppBuilder app, Container container)
+        {
+            // Create an OWIN middleware to create an execution context scope
+            app.Use(async (context, next) =>
```

Comment:
is registered as 'Web API Request' lifestyle, but the instance is requested outside the context of a Web API Request.